juniper_official/Solutions/Krt-async-stuck/krtasync.playbook

```diff
@@ -0,0 +1,156 @@
+This playbook is used to detect KRT Async queue stuck condition and raise alarm.
```

Comment:
461
Entering configuration mode
The configuration has been changed but not committed
|/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:1:(4) syntax error: This
/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:156:(5) error recovery ignores input until this point: }
load complete (2 errors)